demo-vue/src/mixins/vueStricklandMixinFactory.js

```diff
@@ -0,0 +1,178 @@
+import validate from 'strickland';
```

Comment:
Similar to the subtle distinction mentioned above, I'd like to rename this to `vueValidationMixinFactory` and remove "Strickland" from the name.

The spirit of this thinking is that Strickland does not dictate any specific user experience choices or interaction specifics.  Applications can choose the patterns that work for them and use Strickland under the covers.  So these interactions that we've defined for the demo are in place _for this demo only_, and not something that **Strickland** is providing.

Reply:
See above.